replication_parameters.go

```diff
@@ -78,3 +81,24 @@ func (rp ReplicationParameters) getSourceChangesFeedUrl(p ChangesFeedParams) str
 	return changesFeedUrl
 
 }
+
+// sliceEquals returns true if the elements in a exactly match those in b, ignoring the order of elements.
+func sliceEquals(a, b []string) bool {
```

Comment:
what about renaming this to `stringSliceEquals()`

also, I wonder if we need a separate repo like "go-utils" for these kind of super-generic code snippets .. or just use one that's out there and light-weight.

Reply:
I agree, this could probably be renamed to stringSliceEqualsIgnoreOrder(), since that's what's really going on.  We discussed the 'general util' topic a bit, and decided that if this was going into a util library somewhere that it would need to be revisited for performance, but that it wasn't required for this usage.